pages/community/ambassador.js

```diff
@@ -0,0 +1,210 @@
+import GenericLayout from "../../components/layout/GenericLayout";
+import ambassadorsList from "../../config/TSC_MEMBERS.json";
+import TwitterSVG from "../../components/icons/twitter_2";
+import GithubSVG from "../../components/icons/github_2";
+import LinkedInSVG from "../../components/icons/linkedin_2";
+import {sortBy} from 'lodash';
+
+function addAdditionalUserInfo(user) {
+  const userData = {
+    ...user,
+  };
+
+  // if username is not present, use the github username
+  if (!userData.name) userData.name = userData.github;
+
+  // add social links
+  if (userData.github)
+    userData.github = `https://www.github.com/${userData.github}`;
+  if (userData.linkedin)
+    userData.linkedin = `https://www.linkedin.com/in/${userData.linkedin}`;
+  if (userData.twitter)
+    userData.twitter = `https://www.twitter.com/${userData.twitter}`;
+
+  // add avatar url
+  // github redirects to avatar url using `https://www.github.com/<username>.png`
+  userData.avatarUrl = userData.github + ".png";
+
+  // make repo links
+  userData.repos = userData.repos.map((repoName) => ({
+    name: repoName,
+    url: `https://www.github.com/asyncapi/${repoName}`,
+  }));
+
+  return userData;
+}
+
+export default function ambassadors() {
+  const description =
+    "Meet the current AsyncAPI ambassadors and learn how you can become one.";
+  const image = "/img/social/website-card.jpg";
+
+  const ambassadors = sortBy(ambassadorsList.map((user) => addAdditionalUserInfo(user)),['name']);
+
+  return (
+    <GenericLayout
+      title="AsyncAPI ambassadors"
+      description={description}
+      image={image}
+      wide
+    >
+      <div className="py-12 relative max-w-xl mx-auto px-4 sm:px-6 lg:px-8 lg:max-w-screen-xl">
+        <div className="grid lg:grid-cols-3 lg:gap-8">
+          <div>
+            <h3 className="font-semibold  text-primary-800 mb-2 lg:text-2xl lg:text-center">
+            Who is an ambassador?
+            </h3>
+            <p className="my-4 text-base text-gray-500 lg:text-center">
+            The AsyncAPI Ambassador brings AsyncAPI closer to the users and potential users of the project. Provide users and community members with the necessary tools and resources:
+They promote cooperation between users and disseminate the necessary tools to any specification user.
+They are engaged with the project in some way, either as a contributor, blogger, speaker, etc.
+They should be willing to speak at community events or write technical content, such as blog posts, solving community needs.
+            </p>
+          </div>
+          <div>
+            <h3 className="font-semibold  text-primary-800 mb-2 lg:text-2xl lg:text-center">
+            How to become an ambassador?
```

Comment:
also please make a list and link to Code of Conduct + link to guide on how to become an ambassador. I know PR in `community` repo is not merged yet, but link is already known